lib/src/config.rs

```diff
@@ -224,24 +224,24 @@ pub struct Curve25519 {
     pub nonce: secretbox::Nonce,
 }
 
-#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
-/// Chunking Algorithms supported by rdedup
-pub enum ChunkingAlogrithm {
-    Bup,
+#[derive(Copy, Clone, Debug, Serialize, Deserialize, PartialEq)]
+/// ```ChunkingAlgorithm``` are the algorithms supported by rdedup
+pub enum ChunkingAlgorithm {
+    /// ```Bup``` is the default algorithm, the u32 value provided with bup is the bit mask size to
+    /// be used by rollsum. The valid range is between 10 and 30 (1KB to 1GB)
+    Bup(u32),
```

Comment:
So instead of:
```Rust
Bup(u32)
```

You want it to be:
```Rust
Bup {chunk_bits: u32}
```

I think chunk_bits is better for Bup as it describes exactly the value and it won't be confusing if we add another later like so:
```Rust
Fixed {chunk_size: u64}
```